pkg/diff/sql_generator.go

```diff
@@ -49,19 +49,23 @@ var (
 		Type:    MigrationHazardTypeAcquiresAccessExclusiveLock,
 		Message: "Index drops will lock out all accesses to the table. They should be fast",
 	}
+	migrationHazardSequenceCannotTrackDependencies = MigrationHazard{
+		Type:    MigrationHazardTypeHasUntrackableDependencies,
+		Message: "sequence has no owner, so it can't be tracked. It may be in use by a table or function",
+	}
 )
 
-type oldAndNew[S schema.Object] struct {
+type oldAndNew[S any] struct {
 	old S
 	new S
 }
 
 func (o oldAndNew[S]) GetNew() S {
-	return o.old
+	return o.new
 }
 
 func (o oldAndNew[S]) GetOld() S {
-	return o.new
+	return o.old
```

Comment:
Crazy bug. I'm shocked this was working, but I'm guessing it's because we only use this functions in a few places